src/vorpal.ts

```diff
@@ -898,22 +890,25 @@ export default class Vorpal extends EventEmitter implements IVorpal {
 
       // If we get a string back, it's a validation error.
       // Show help and return.
-      if (_.isString(item.args) || !_.isObject(item.args)) {
+      if (isString(item.args) || !isObject(item.args)) {
         throwHelp(item, item.args);
         return callback(item, undefined, item.args);
       }
 
       // Build the piped commands.
       let allValid = true;
       for (let j = 0; j < item.pipes.length; ++j) {
-        const commandParts = self.util.matchCommand(item.pipes[j], self.commands);
+        const commandParts = utils.matchCommand(item.pipes[j], self.commands);
         if (!commandParts.command) {
           item.session.log(self._commandHelp(item.pipes[j]));
           allValid = false;
           break;
         }
-        commandParts.args = self.util.buildCommandArgs(commandParts.args, commandParts.command);
-        if (_.isString(commandParts.args) || !_.isObject(commandParts.args)) {
+        commandParts.args = (utils.buildCommandArgs(
+          commandParts.args,
+          commandParts.command
+        ) as unknown) as string;
```

Comment:
feels a bit weird that double cast. Seems a bad smell to me
(or a sign I indeed need raising my typing level :))